.travis.yml

```diff
@@ -0,0 +1,39 @@
+sudo: true
+
+language: python
+
+services:
+  - docker
+
+env:
+  -
+    - TOXENV=py27
```

Comment:
Splitting up TOX into multiple environmental variables like this will incur a lot of overhead. We should put this inside the .test. Same with PYTHON_MAJOR (unless there is some valid reason to keep it).